extensions/notebook/src/common/localizedConstants.ts

```diff
@@ -21,8 +21,10 @@ export const confirmReplace = localize('confirmReplace', "Folder already exists.
 export const openNotebookCommand = localize('openNotebookCommand', "Open Notebook");
 export const openMarkdownCommand = localize('openMarkdownCommand', "Open Markdown");
 export const openExternalLinkCommand = localize('openExternalLinkCommand', "Open External Link");
-
+export const msgBookTrusted = localize('msgBookTrusted', "Book is now automatically trusted.");
```

Comment:
Can we change this string from "automatically" to something that includes "workspace"